tests/test_modeling_flax_common.py

```diff
@@ -195,19 +237,47 @@ def test_jit_compilation(self):
 
                 @jax.jit
                 def model_jitted(input_ids, attention_mask=None, token_type_ids=None):
-                    return model(input_ids, attention_mask, token_type_ids)
+                    return model(
+                        input_ids=input_ids,
+                        attention_mask=attention_mask,
+                        token_type_ids=token_type_ids,
+                    ).to_tuple()
+
+                with self.subTest("JIT Enabled"):
+                    jitted_outputs = model_jitted(**prepared_inputs_dict)
 
                 with self.subTest("JIT Disabled"):
                     with jax.disable_jit():
                         outputs = model_jitted(**prepared_inputs_dict)
 
-                with self.subTest("JIT Enabled"):
-                    jitted_outputs = model_jitted(**prepared_inputs_dict)
-
                 self.assertEqual(len(outputs), len(jitted_outputs))
                 for jitted_output, output in zip(jitted_outputs, outputs):
                     self.assertEqual(jitted_output.shape, output.shape)
 
+                @jax.jit
+                def model_jitted_return_dict(input_ids, attention_mask=None, token_type_ids=None):
+                    return model(
+                        input_ids=input_ids,
+                        attention_mask=attention_mask,
+                        token_type_ids=token_type_ids,
+                    )
+
+                # jitted function cannot return OrderedDict
+                with self.assertRaises(TypeError):
```

Comment:
Jitted function cannot return OrderedDict => this shouldn't really be a problem though